docs/content/latest/deploy/kubernetes/single-zone/gke/helm-chart.md

```diff
@@ -70,11 +70,8 @@ $ gcloud config set compute/zone us-west1-b
 
 - Install `kubectl`
 
-After installing Cloud SDK, install the `kubectl` command line tool by running the following command.
+Refer to your specific [operating system](https://kubernetes.io/docs/tasks/tools/) for kubectl install.
```

Comment:
This doc is specific to GKE, though. Why remove this? I know you commented in the PR description that the path may vary, but it seems like that means we should just point out what the options are here, not punt to the k8s docs.

Reply:
When someone installs kubectl on their machine, typically they would follow the instruction to install one kubectl cli, common for all k8s use cases across all clouds and on-prem. You will not have one kubectl for gcloud and another for aws and another for azure. 

So following the [doc](https://kubernetes.io/docs/tasks/tools/#kubectl) for example if using mac, you could use homebrew to install it. Or for windows, chocolatey ( similar to homebrew ) is another option. If  it is installed to a homebrew location then we do not need a kubectl installation under the gcloud install directory. 

For example on my mac,  I have kubectl in two places

- /opt/homebrew/bin/kubectl
- /opt/google-cloud-sdk/bin/kubectl

But since my $PATH variable lists /opt/homebrew/bin before /google-cloud-sdk/bin, when I type kubectl it runs the one under /opt/homebrew only. So the one one under gcloud diorectory is not used ( unless specify the full path )

Now the gcloud itself is installed now using homebrew, but using separate [install](https://cloud.google.com/sdk/docs/install) . Part of the install lets you specify which you want to gcloud to install.

We probably do not get too much into gcloud install ( :-) after all we are cloud native )..

Reply:
Makes sense!